bigquery_etl/cli/metadata.py

```diff
@@ -72,5 +72,4 @@ def update(name: str, sql_dir: Optional[str], project_id: Optional[str]) -> None
 
         dataset_metadata.write(dataset_metadata_path)
         table_metadata.write(table_metadata_file)
-        click.echo(f"Updated {table_metadata_file}")
```

Comment:
I'm guessing you're removing this because it's saying basically every one is being updated?

Instead of removing the logging IMO it'd be better to improve this logic to keep track of whether the metadata was actually updated, and then only write the files and log when they actually were (which should also be a bit faster since fewer files would be written).
```python
        dataset_metadata_updated = False
        table_metadata_updated = False

        # set dataset metadata default_table_workgroup_access to table_workgroup_access if not set
        if not dataset_metadata.default_table_workgroup_access:
            dataset_metadata.default_table_workgroup_access = (
                dataset_metadata.workgroup_access
            )
            dataset_metadata_updated = True

        if table_metadata.deprecated:
            # set workgroup: [] if table has been tagged as deprecated
            # this overwrites existing workgroups
            if table_metadata.workgroup_access:
                table_metadata.workgroup_access = []
                table_metadata_updated = True
            if dataset_metadata.workgroup_access:
                dataset_metadata.workgroup_access = []
                dataset_metadata_updated = True
        elif table_metadata.workgroup_access is None:
            table_metadata.workgroup_access = (
                dataset_metadata.default_table_workgroup_access
            )
            table_metadata_updated = True

        if dataset_metadata_updated:
            dataset_metadata.write(dataset_metadata_path)
            click.echo(f"Updated {dataset_metadata_path}")
        if table_metadata_updated:
            table_metadata.write(table_metadata_file)
            click.echo(f"Updated {table_metadata_file}")
```